### ISSUE TITLE
addTestCaseKeywords and removeTestCaseKeywords api calls are not working.

### ISSUE THREAD

**Issue by dunnap** [lczub/TestLink-API-Python-client#64]
```
response = mytestlink.addTestCaseKeywords({'TC-211026':['Sanity']})
```

  File "C:\Python27\lib\site-packages\testlink\testlinkdecorators.py", line 112, in wrapperAddDevKey
    return methodAPI(self, _argsPositional, *_argsOptional)
  File "C:\Python27\lib\site-packages\testlink\testlinkdecorators.py", line 100, in wrapperWithArgs
    _argsPositional, *_argsOptional)
  File "C:\Python27\lib\site-packages\testlink\testlinkapigeneric.py", line 1320, in callServerWithPosArgs
    self._checkResponse(response, methodNameAPI, argsOptional) 
  File "C:\Python27\lib\site-packages\testlink\testlinkapigeneric.py", line 1587, in _checkResponse
    'Empty Response! ')
testlink.testlinkerrors.TLResponseError: Empty Response! 
addTestCaseKeywords({'keywords': {'TC-211026': ['Sanity']}, 'devKey': 'd28c00d400b1ea36be53515d65dbde88'})

**Comment by lczub** [comment]
Hello dunnap,

I'm sorry, but I can not reproduce your failure in my environment TestLink Server 1.9.14 plus TestLink-API-Python-client 0.6.2. Which combination do you use?

These are my test variation.

pos. Test addTestCaseKeywords response  not yet assigned known keyword:
- returns {'status_ok': True, 'validKeywords': {'NPROAPI34-2': {'34': 'KeyWord01'}}}

neg. Test addTestCaseKeywords response already assigned keyword
- returns {'validKeywords': {'NPROAPI34-2': {'34': 'KeyWord01'}}, 'status_ok': True}

neg. Test addTestCaseKeywords Response unknown external ID: 
- returns testlink.testlinkerrors.TLResponseError: 5040: (addTestCaseKeywords) - Test Case External ID (TC-211026) does not exist!

neg. Test addTestCaseKeywords Response unknown Keyword: 
- returns {'validKeywords': {'NPROAPI34-2': ''}, 'status_ok': False}

pos. Test removeTestCaseKeywords response currently assigned (known) keyword:
- returns {'status_ok': True, 'validKeywords': {'NPROAPI34-2': {'34': 'KeyWord01'}}}

neg. Test removeTestCaseKeywords response already removed (or not assigned) known keyword:
- returns {'validKeywords': {'NPROAPI34-2': {'35': 'KeyWord02'}}, 'status_ok': True}

neg. Test removeTestCaseKeywords Response unknown external ID: 
- returns testlink.testlinkerrors.TLResponseError: 5040: (removeTestCaseKeywords) - Test Case External ID (TC-211026) does not exist!

neg. Test removeTestCaseKeywords Response unknown Keyword: 
- returns {'status_ok': False, 'validKeywords': {'NPROAPI34-2': ''}}

Regards Luiko

**Comment by dunnap** [comment]
Hi Luiko,

My environment is TestLink Server 1.9.13 (Stormbringer) and
TestLink-API-Python-client 0.6.2.

response = mytestlink.addTestCaseKeywords({'TC-211026':['Sanity']})
print (response)


Thanks,
Pradeep.

**Comment by lczub** [comment]
Hi Pradeep,
sorry, I find currently no time to switch my environment back to TL 1.9.13.
Maybe next week. Is there anyone else, who can reproduce this problem?

Regards luiko

**Comment by lczub** [comment]
Hello Pradeep
do you still have this problem or is it fixed with TL 1.9.14/15 ?
Regards Luiko
